DicomRtImportExport/Logic/vtkSlicerDicomRtReader.cxx

```diff
@@ -103,7 +138,7 @@ class vtkSlicerDicomRtReader::vtkInternal
     unsigned int Number;
     std::string Name;
     std::string Description;
-    double DisplayColor[3];
+    std::array< double, 3 > DisplayColor;

```

Comment:
I believe we used a static array because of VTK set/get macros. Is there a good reason to change this?

Reply:
There is only one method which use DisplayColor
`double* vtkSlicerDicomRtReader::GetRoiDisplayColor(unsigned int internalIndex)`.
The return value is a pointer, so no difference. Everything else is a DCMTK specific.

No setters to change a display color, within internal elements of the reader.

I that case i had thought it's better to use std::array, rather C array (no copy, assign problems).

I can roll it back, if its required to use C array.

Reply:
If it's only internal then it's fine like this, thanks!